### DISCUSSION TITLE
Tiled sprite like cocos creator

### DISCUSSION THREAD

**Discussion by antn9x** [axmolengine/axmol#1007] [General]
How to render sprite as tiled in cocos creator?
```
Texture2D::TexParams params = {SamplerFilter::LINEAR, SamplerFilter::LINEAR, SamplerAddressMode::REPEAT, SamplerAddressMode::REPEAT};
sprite->getTexture()->setTexParameters(params);
```
I used this code but not working.

**Comment by rh101** [comment]
There should be examples in the cpp-tests project, which should always be your first point of reference.  If something is related to Cocos Creator, then best to ask on the cocos forums (where I see you've already done so).

**Reply by antn9x** [reply]
I found this [snippet code](https://github.com/cocos2d/creator_to_cocos2dx/blob/32aae979263570c2fc69c9074519e59a501326b6/creator_project/packages/creator-luacpp-support/reader/CreatorReader.cpp#L1480 ), but it worked with axmol, please help me to make it work.

**Reply by rh101** [reply]
Look, I'm going to be quite blunt.

- You haven't stated what the problem is.
- You haven't shown what attempts you have made yourself to solve whatever problem you are having. such as your own code or any detailed information related to the issue.  You just seem to be posting random snippets of what seems to be code written by others, and asking other people to use their time and effort to do the work for you, which is completely unreasonable.

So, if you provide a detailed explanation of what the problem is, and show what attempts you have made at solving it, then there is a chance that someone may be able to help you find a solution.

**Reply by antn9x** [reply]
you are right. this is my problem and i have to fix it. I just don't know start from where for what i un know. then i ask for any help, snippet code, documents, guide, keywords. or ask for someone have the same problem and fixed it.

**Reply by rh101** [reply]
You still seem to be missing the point.  You need to explain what the actual issue is; what is it that you are trying to achieve? Be detailed, and show images of the input and the expected output if you can.

**Reply by antn9x** [reply]
Hi, after fix code order, then I can get this. Bug fixed.
